plugins/docsettingtweak.koplugin/directory_defaults.lua

```diff
@@ -0,0 +1,33 @@
+return {--do NOT change this line
+
+ --HELP:
+ -- you can override default settings for documents per directory in this file
+ -- the directories must be under the home folder.
+ -- you can find settings to change in the metadata.lua in the .sdr folder.
+
+ -- syntax:
+
+ --   ["path/to/folder"] = {
+ --         ["setting_to_override"] = value,
+ --    },
+
+ -- examples:
+
+ --[[
+    ["/mnt/us/documents/hebrew"] = {
+        ["inverse_reading_order"] = true
+    },
+    ["/mnt/us/documents/smalltext"] = {
+        ["font_size"] = 34,
+        ["line_space_percent"] = 130,
+    },
+--]]
+
+ -- remember to put coma at the end of each line!
```

Comment:
Comments like to be proper sentences.
```suggestion
-- Remember to put a comma at the end of each line!
```